litgpt/finetune/full.py

```diff
@@ -54,7 +52,24 @@ def setup(
         max_seq_length=None,
     ),
     eval: EvalArgs = EvalArgs(interval=600, max_new_tokens=100, max_iters=100),
+    logger_name: Literal["wandb", "tensorboard", "csv"] = "csv",
+    seed: int = 1337,
 ) -> None:
+    """Finetune a model.
+
+    Arguments:
+        checkpoint_dir: The path to the base model checkpoint directory to load for finetuning.
+        out_dir: Where to save checkpoints and logs.
+        precision: The precision to use for finetuning. Possible choices: "bf16-true", "bf16-mixed", "32-true".
+        devices: How many devices/GPUs to use
+        resume: Path to a checkpoint directory to resume from in case training got interrupted. Or `True` to resume
+            from the latest checkpoint in `out_dir`.
+        data: Data related arguments. If not provided, the default is `litgpt.data.Alpaca`.
+        train: Training related arguments. See `litgpt.args.TrainArgs` for details.
```

Comment:
We are not generating sphinx docs but maybe we still want to use the sphinx code notation for consistency?

```suggestion
        train: Training related arguments. See ``litgpt.args.TrainArgs`` for details.
```

Reply:
Yes can do. The only thing is that it will also show like this in the help page, so might look a bit weird.

Reply:
Which way do you lean? I'm fine with either